sdk/src/test/java/com/qonversion/android/sdk/internal/userProperties/UserPropertiesStorageImplTest.kt

```diff
@@ -303,15 +328,16 @@ internal class UserPropertiesStorageImplTest {
         fun `delete existing property`() {
             // given
             val key = "keyToDelete"
+            val value = "value"
 
-            val existingProperties = mutableMapOf("keyToDelete" to "value")
+            val existingProperties = mutableMapOf(key to value)
```

Comment:
Let's add extra properties to `existingProperties`  to be sure that the `delete` func deletes only one pair, but not clears map.